pywbemtools/pywbemcli/_cmd_profile.py

```diff
@@ -0,0 +1,252 @@
+                       options_metavar=CMD_OPTS_TXT)
+@click.option('-o', '--organization', type=str, metavar='ORG-NAME',
+              required=False,
+              help='Filter by the defined organization. (ex. -o DMTF')
+@click.option('-p', '--profile', type=str, metavar='PROFILE-NAME',
+              required=False,
+              help='Filter by the profile name. (ex. -p Array')
+@add_options(help_option)
+@click.pass_obj
+def profile_list(context, **options):
+    """
+    List management profiles advertised by the server.
+
+    Retrieve the CIM instances representing the WBEM management profiles
+    advertized by the WBEM server, and display information about each profile.
+    WBEM management profiles are defined by DMTF and SNIA and define the
+    management functionality that is available.
+
+    The retrieved profiles can be filtered using the --organization and
+    --profile options.
+
+    The output is formatted as a table showing the organization, name, and
+    version for each profile. The --output-format option is ignored unless it
+    specifies a table format.
+    """
+    context.execute_cmd(lambda: cmd_profile_list(context, options))
+
+
+@profile_group.command('centralinsts', cls=PywbemcliCommand,
+                       options_metavar=CMD_OPTS_TXT)
+@click.option('-o', '--organization', type=str, metavar='ORG-NAME',
+              required=False,
+              help='Filter by the defined organization. (ex. -o DMTF')
+@click.option('-p', '--profile', type=str, metavar='PROFILE-NAME',
+              required=False,
+              help='Filter by the profile name. (ex. -p Array')
+@click.option('--cc', '--central-class', 'central_class', type=str,
+              metavar='CLASSNAME', required=False,
+              help='Optional. Required only if profiles supports only '
+              'scopig methodology')
+@click.option('--sc', '--scoping-class', 'scoping_class', type=str,
+              metavar='CLASSNAME', required=False,
+              help='Optional. Required only if profiles supports only '
+              'scopig methodology')
+@click.option('--sp', '--scoping-path', 'scoping_path', type=str,
+              metavar='CLASSLIST', required=False, multiple=True,
+              help='Optional. Required only if profiles supports only '
+              'scopig methodology. Multiples allowed')
+@click.option('--rd', '--reference-direction', 'reference_direction',
+              type=click.Choice(['snia', 'dmtf']),
+              default='dmtf',
+              show_default=True,
+              help='Navigation direction for association.')
+@add_options(help_option)
+@click.pass_obj
+def centralinsts(context, **options):
+    """
+    List central instances of mgmt profiles on the server.
+
+    Retrieve the CIM instances that are central instances of the specified
+    WBEM management profiles, and display these instances. By default, all
+    management profiles advertized on the server are used. The profiles
```

Comment:
Change "are used" to something else, e.g. "are included".

Reply:
DONE